src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedSubstitutedTypeParameterSymbol.cs

```diff
@@ -34,6 +34,19 @@ public override bool IsImplicitlyDeclared
 
         internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<CSharpAttributeData> attributes)
         {
+            if (ContainingSymbol.Kind == SymbolKind.NamedType &&
+                _underlyingTypeParameter.OriginalDefinition is SourceMethodTypeParameterSymbol &&
```

Comment:
>is SourceMethodTypeParameterSymbol [](http://example.com/codeflow?start=60&length=34)

Consider capturing in a local and reusing below: `is SourceMethodTypeParameterSymbol underlyingDefinition`

Reply:
> > is SourceMethodTypeParameterSymbol
> 
> Consider capturing in a local and reusing below: `is SourceMethodTypeParameterSymbol underlyingDefinition`

I'll make this adjustment in the next PR